### PR TITLE
Cleanup codestarter test

### PR DESCRIPTION
**Pull request by turing85**
- Remove unnecessary modifiers
- Unify  line-breaks 
- Unify indentation

### CODE REVIEW

**Review by iocanel — APPROVED** [review]
LGTM

**Comment by turing85** [comment]
uhm... I am a little bit at a loss here. Tests fail with
```
java.lang.AssertionError: 
[Snapshot is not matching (use -Dsnap to update it automatically): GrpcCodestartTest/testContent/src_test_java_ilove_quark_us_HelloGrpcServiceTest.java] 
```

Is this ignorable? If not: how to fix this?

**Comment by ia3andy** [comment]
@turing85 you need to either:
- Fix the snapshots manually
- Run the test with -Dsnap to update them (as explained in the log)

The snapshots are here to make sure the result of generating app with your changes in the templates is as expected

**Comment by gsmet** [comment]
I pushed an update to synchronize the snapshots.

File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-routes-codestart/java/src/test/java/org/acme/MyDeclarativeRoutesTest.java [hunk 1]
```diff
@@ -8,15 +8,16 @@
...
-	}
+		@Test
+		void testHelloRouteEndpointWithNameParameter() {
+				given().when().get("hello-route?name=Quarkus").then().statusCode(200).body(is("Hello Quarkus !!"));
+		}
+
+		@Test
+		void testHelloRouteEndpointWithoutNameParameter() {
+				given().when().get("hello-route").then().statusCode(200).body(is("Hello Reactive Route !!"));
+		}
```
Comment (gsmet) [inline devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-routes-codestart/java/src/test/java/org/acme/MyDeclarativeRoutesTest.java:21]: I think you are using tabs here? At least, that's what GitHub seems to show? We should use 4 spaces instead.

**Review by gsmet — DISMISSED** [review]
I'm not sure about how two files are indented. Could you have a look?

**Comment by turing85** [comment]
> I'm not sure about how two files are indented. Could you have a look?

The tabs were there already. I replaced them. I also rebased onto the current main. @gsmet care to re-evalute the PR now?

By-the-by: I did a global search on tabs in `*.java` files. Let's say there were enough hits that intelliJ refused to replace them all in one go. Should we create an issue to replace all those tabs?

**Comment by turing85** [comment]
so... we wanna proceed with this PR or nah?

File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-routes-codestart/java/src/test/java/org/acme/MyDeclarativeRoutesTest.java (hunk 1 above)
Comment (turing85) [reply devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-routes-codestart/java/src/test/java/org/acme/MyDeclarativeRoutesTest.java]: Fixed

File: integration-tests/devtools/src/test/resources/__snapshots__/ReactiveRoutesCodestartTest/testContent/src_test_java_ilove_quark_us_MyDeclarativeRoutesTest.java [hunk 2]
```diff
@@ -8,15 +8,16 @@
...
-	}
+		@Test
+		void testHelloRouteEndpointWithNameParameter() {
+				given().when().get("hello-route?name=Quarkus").then().statusCode(200).body(is("Hello Quarkus !!"));
+		}
+
+		@Test
+		void testHelloRouteEndpointWithoutNameParameter() {
+				given().when().get("hello-route").then().statusCode(200).body(is("Hello Reactive Route !!"));
+		}
```
[the remark this answers is not in the document]
Comment (turing85) [reply integration-tests/devtools/src/test/resources/__snapshots__/ReactiveRoutesCodestartTest/testContent/src_test_java_ilove_quark_us_MyDeclarativeRoutesTest.java]: Fixed

**Review by ia3andy — APPROVED** [review]
LGTM

**Pull request merged**